invenio_accounts/templates/semantic-ui/invenio_accounts/login_user.html

```diff
@@ -10,45 +10,48 @@
 {% from "invenio_accounts/_macros.html" import render_field, form_errors %}
 
 {% block page_body %}
-<div class="ui stacked padded segments big form">
-  <div class="ui segment">
+  <div class="ui stacked padded segments big form">
+    <div class="ui segment padded">
+      <div class="divider hidden"></div>
+      {%- block form_header %}
+        <h3 class="ui header">{{ _('Log in to account') }}</h3>
+      {%- endblock form_header %}
+
+      {%- block form_outer %}
+        {%- with form = login_user_form %}
+          <form action="{{ url_for_security('login') }}" method="POST"
+                name="login_user_form">
+            {{ form.hidden_tag() }}
+            {{ form_errors(form) }}
+            {{ render_field(form.email, icon="user icon", autofocus=True, errormsg=False) }}
+            {{ render_field(form.password, icon="lock icon", errormsg=False) }}
+            <button type="submit" class="ui fluid large submit primary button">
+              <i class="ui sign-in icon"></i>{{ _('Log In') }}
```

Comment:
super minor: WDYT?
I know it was like this, but I was reading for ILS that it is preferable the Sign In form... plus I see Sign up below?
```suggestion
              <i class="ui sign-in icon"></i>{{ _('Sign In') }}
```
And maybe to change it also in the header to keep it consistent?